src/lib/components/workflow/dropdown-filter/workflow-type.svelte

```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+  import DropdownMenu from '$lib/components/dropdown-menu.svelte';
+  import Input from '$lib/holocene/input/input.svelte';
+  import { workflowFilters, workflowSorts } from '$lib/stores/filters';
+  import { updateQueryParamsFromFilter } from '$lib/utilities/query/to-list-workflow-filters';
+
+  let value = '';
+
+  const getOtherFilters = () =>
+    $workflowFilters.filter((f) => f.attribute !== 'WorkflowType');
+
+  $: typeFilter = $workflowFilters.find((f) => f.attribute === 'WorkflowType');
+  $: typeSort = $workflowSorts.find((s) => s.attribute === 'WorkflowType');
+
+  const onInput = (e: Event) => {
+    const { value } = e.target as HTMLInputElement;
+    if (value) {
+      const filter = {
+        attribute: 'WorkflowType',
+        value,
+        conditional: '=',
+        operator: '',
+        parenthesis: '',
+      };
+      $workflowFilters = [...getOtherFilters(), filter];
+    } else {
+      $workflowFilters = [...getOtherFilters()];
+    }
+
+    updateQueryParamsFromFilter($page.url, $workflowFilters, $workflowSorts);
```

Comment:
I noticed that it was taking me a while to type out a workflow type and it doesn't do partial matching so it looks like there are no matching workflows until you type out the whole thing. 

Maybe we could add `debounce` here to give the user a chance to type out the full workflow? e.g.

```js
  const update = debounce(updateQueryParamsFromFilter, 800);
```

Reply:
We debounce with 300, but it does seem a little too quick

Reply:
Ah now, I see it on the `FilterInput` 🤦‍♀️ Yeah, maybe just making that longer? Although I know 300 is pretty standard...